cloud_experts_tutorials/cloud-experts-dynamic-certificate-custom-domain.adoc

```diff
@@ -0,0 +1,522 @@
+:_content-type: ASSEMBLY
+[id="cloud-experts-dynamic-certificate-custom-domain"]
+= Tutorial: Dynamically issue certificates using the cert-manager Operator on ROSA
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-dynamic-certificate-custom-domain
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-20
+//---
+//date: '2022-10-11'
+//title: Dynamic Certificates for ROSA Custom Domain
+//weight: 1
+//tags: ["AWS", "ROSA"]
+//authors:
+//   Kevin Collins
+//---
+
+While wildcard certificates provide simplicity by securing all first-level subdomains of a given domain with a single certificate, other use cases may require the use of individual certificates per domain. This tutorial guides you through using the link:https://docs.openshift.com/container-platform/latest/security/cert_manager_operator/index.html[cert-manager Operator for Red Hat OpenShift] and link:https://letsencrypt.org/[Let's Encrypt] to dynamically issue certificates for routes created using a custom domain. 
+
+== Prerequisites
+
+* A ROSA cluster
+* You have access to the OpenShift CLI (`oc`)
+* You have access to the AWS CLI (`aws`)
+* A unique domain, such as *.apps.<company_name>.io
+* An Amazon Route 53 public hosted zone for the above domain
+
+=== Environment setup
```

Comment:
I'm not sure if this should be a level three or level two subheading. It isn't a crucial debate however and we can always adjust it later.